src/pages/WriteProductsPage.tsx

```diff
@@ -9,59 +10,72 @@ type FormData = {
 
 export const WriteProductsPage: React.FC = () => {
 	const { register, handleSubmit, formState: { errors } } = useForm<FormData>();
+	const [text, setText] = useState('投稿する？');
```

Comment:
```suggestion
	const [text, setText] = useState('');
```
こんな感じでも大丈夫かな？　感覚だけど、useState の初期値はundefinedとか0とかnullとか、値がない系のものが多い印象がある。